.circleci/config.yml

```diff
@@ -90,43 +104,42 @@ jobs:
       - run: echo 'export GITHUB_ORGANIZATION=$CIRCLE_PROJECT_USERNAME' >> $BASH_ENV
       - run: echo 'export GITHUB_REPOSITORY=$CIRCLE_PROJECT_REPONAME' >> $BASH_ENV
       - run: echo 'export DOCKER_BASE_TAG=$CIRCLE_TAG' >> $BASH_ENV
+      - oss-docs/install-tools
+      - rok8s/get_vault_env:
+          vault_path: repo/global/env
       - *build_docker_tags
       - *docker_login
       - *docker_build
       - *docker_push
 
+  publish-dev-orb:
+    working_directory: /go/src/github.com/fairwindsops/rok8s-scripts
+    docker:
+      - image: quay.io/reactiveops/ci-images:v11-buster
+    shell: /bin/bash
+    steps:
+      - checkout
+      - oss-docs/install-tools
+      - rok8s/get_vault_env:
+          vault_path: repo/global/env
+      - *install_circleci
+      - orb-tools/pack:
+          source: orb/
+      - orb-tools/publish:
+          orb-ref: fairwinds/rok8s-scripts@dev:${CIRCLE_BRANCH}
+          token-variable: CIRCLECI_DEV_API_TOKEN
+
   request-orb-publish:
     docker:
-      - image: alpine:latest
+      - image: ubuntu:slim
```

Comment:
interesting - can we not require this executor from the oss-docs orb?

Reply:
This has nothing to do with that orb, so I'm really confused.